1MI0800224/week2/week2_task3.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <fstream>
+struct pair{
+    int x;
+    int y;
+};
+
+pair createPair(int x, int y){
+    pair newPair={x, y};
+    return newPair;
+}
+
+void InitPair(pair& toInit, int x, int y){
+    toInit={x, y};
+}
+
+pair ReadPairFromFile(std::ifstream& inFile){
+    if(!inFile.is_open()){
+        return {INT_MIN, INT_MAX};

```

Comment:
You can check [this one](https://www.tutorialspoint.com/cplusplus/cpp_exceptions_handling.htm) too :P